content/docs/user-guide/networking/multiplayer/configuration.md

```diff
@@ -5,7 +5,65 @@ description: Learn how to add multiplayer support to an Open 3D Engine (O3DE) pr
 weight: 200
 ---
 
-Adding the full functionality of the **Multiplayer Gem** to an **Open 3D Engine (O3DE)** project requires making edits to the CMake scripts and source code. These changes enable:
+When you need multiplayer support in your **Open 3D Engine (O3DE)** project, the easiest way to set it up is to create the project from the O3DE multiplayer template using the instructions that follow.
+
+If your O3DE project is already in development and you want to add multiplayer support to it, the instructions on [Manual project configuration](#manual-project-configuration) will help you add the **Multiplayer Gem** to an existing project.
+
+## Creating a multiplayer project from the multiplayer template {#creating-project-from-template}
+
+To create a multiplayer project from the O3DE multiplayer template, you need O3DE version 22.10 or later. You must first add the template from the [O3DE Extras GitHub repo](https://github.com/o3de/o3de-extras) to your library of O3DE templates using **Project Manager**. Then you will be able to create a new project from the template.
+
+1. In Project Manager, choose **New Project**, then **Create New Project**.
```

Comment:
Good question. I think if a reader needs help creating a new project, it would be best to link them to the instructions for Project Manager on how to create a project. I'll add that in a follow-on PR. Thanks!